ardupilot/source/index.rst

```diff
@@ -26,9 +27,6 @@
     operators. ArduPilot has a huge online community dedicated to
     helping users with questions, problems, and solutions </p>
 
-    <p style="text-align:center;">Copter -- Plane -- Rover -- Sub -- Antenna Tracker</p>
```

Comment:
this is a title of the composite photo..why remove?

Reply:
Basically just to make it look clean. I started to add blimp to it but then I realized I did not like the "--" part of the title and thought it looked cleaner removed. Maybe I can put it back under the composite photo? Or do you think it should be on top?

Reply:
I dont have a strong feeling one way or the other